profile/Readme.md

```diff
@@ -5,55 +5,26 @@
+    <a href="https://tkmm.org/docs">
+    <img alt="Documentation" src="https://img.shields.io/badge/Documentation-%23FFFFFF?style=flat&logo=data%3Aimage%2Fsvg%2Bxml%3Bbase64%2CPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjBweCIgdmlld0JveD0iMCAtOTYwIDk2MCA5NjAiIHdpZHRoPSIyMHB4IiBmaWxsPSIjMDAwMDAwIj48cGF0aCBkPSJNMjE2LTE0NHEtMjkuNyAwLTUwLjg1LTIxLjE1UTE0NC0xODYuMyAxNDQtMjE2di01MjhxMC0yOS43IDIxLjE1LTUwLjg1UTE4Ni4zLTgxNiAyMTYtODE2aDIyOHExNS4zIDAgMjUuNjUgMTAuMjlRNDgwLTc5NS40MiA0ODAtNzgwLjIxdC0xMC4zNSAyNS43MVE0NTkuMy03NDQgNDQ0LTc0NEgyMTZ2NTI4aDUyOHYtMjI4cTAtMTUuMyAxMC4yOS0yNS42NVE3NjQuNTgtNDgwIDc3OS43OS00ODB0MjUuNzEgMTAuMzVRODE2LTQ1OS4zIDgxNi00NDR2MjI4cTAgMjkuNy0yMS4xNSA1MC44NVE3NzMuNy0xNDQgNzQ0LTE0NEgyMTZabTUyOC01NDlMNDEyLTM2MXEtMTEgMTEtMjUgMTAuNVQzNjItMzYycS0xMS0xMS0xMS0yNS41dDExLTI1LjVsMzMxLTMzMWgtODFxLTE1LjMgMC0yNS42NS0xMC4yOVE1NzYtNzY0LjU4IDU3Ni03NzkuNzl0MTAuMzUtMjUuNzFRNTk2LjctODE2IDYxMi04MTZoMTY4cTE1LjMgMCAyNS42NSAxMC4zNVE4MTYtNzk1LjMgODE2LTc4MHYxNjhxMCAxNS4zLTEwLjI5IDI1LjY1UTc5NS40Mi01NzYgNzgwLjIxLTU3NnQtMjUuNzEtMTAuMzVRNzQ0LTU5Ni43IDc0NC02MTJ2LTgxWiIvPjwvc3ZnPg%3D%3D">
 </p>
 
-# Download TKMM
-
-* [Windows (x64/arm64)](https://tkmm-team.github.io/Tkmm-Wiki/downloads/index.html?tabs=windows)
-* [Linux (x64/arm64)](https://tkmm-team.github.io/Tkmm-Wiki/downloads/index.html?tabs=linux)
-* [macOS (x64/Apple Silicon)](https://tkmm-team.github.io/Tkmm-Wiki/downloads/index.html?tabs=macos)
-
-# Help & Documentation
-
-[TKMM's Support Website](https://tkmm-team.github.io/Tkmm-Wiki/index.html)
-
-# Discord
-
-[Join our Discord community](https://discord.com/invite/3tNUBWQRXv)
+**TKMM** (**T**ears of the **K**ingdom **M**od **M**anager) is a mod packager and merger for Tears of the Kingdom.
 
-# Written by the TKMM Team
+**TKMM** manages many aspects of mod management, such as RESTBL calculation, overlapping files, changelogs and more, ensuring a streamlined process and reducing hassle when using mods.
 
-* [Arch Leaders](https://github.com/ArchLeaders)
-* [Lord Bubbles](https://github.com/MasterBubbles)
-* [Mikachan](https://github.com/okmika)
-* [Aster](https://github.com/AsteroidPizza39)
-* [The5thTear](https://github.com/The5thTear)
+TKCL's are the format that **TKMM** packages mods into. They support features such as [Option Groups](https://tkmm.org/docs/using-mods/#configuring-options), have smaller file sizes, support changelogs and other features. If you're downloading a mod to merge with **TKMM**, it is reconmended to look for a TKCL file available for the mod first.
```

Comment:
I don't see the purpose of explaining TKCL files on the organization's page. This isn't the documentation

Reply:
This was one of the things I chose when choosing things to talk about when it comes to reasons to use TKMM: but you're right in the way that it doesn't need to be very high detail and can just be a small note, and not emphasized. I will remove it from here, and it should also be added to the documentation.

Reply:
Removed this blurb